### ISSUE TITLE
Enhance a11y for `useInfiniteScroll`

### ISSUE THREAD

**Issue by hirotomoyamada** [yamada-ui/yamada-ui#5399]
name: useInfiniteScroll

### Description
1 accessibility violation was found in the `useInfiniteScroll`.

#### [Hooks / useInfiniteScroll / Root](https://yamada-ui.github.io/yamada-ui/?path=/story/hooks-useinfinitescroll--root)

Received:
Scrollable region must have keyboard access (scrollable-region-focusable)

Fix any of the following:
  Element should have focusable content
  Element should be focusable

You can find more information on this issue here: 
https://dequeuniversity.com/rules/axe/4.11/scrollable-region-focusable?application=axeAPI

### Additional Information
While it's ideal to cover all tests, if there are too many lines to work on, please create a PR within your capacity.
Any deficiencies not covered by the PR will be automatically created as issues.

**Comment by watataku11xx** [comment]
I've been working on this. Could you please confirm?

It looks like I need some Vercel-related approval.
